docs/sphinx/source/changelog/v2.0.1.rst

```diff
@@ -0,0 +1,14 @@
+*************************
+v2.0.1 (October 30, 2020)
+*************************
+
+Deprecations
+------------
+* The the deprecation of :py:func:`~rdtools.normalization.pvwatts_dc_power`,
```

Comment:
Oops, missed double "the" in the review.  Also has an unnecessary comma at the end of this line